website/src/components/Heading.js

```diff
@@ -27,11 +27,12 @@ function Heading() {
   return (
     <HeadingContainer>
       <StyledLink to="/">
-        bedav
+        Bedav
```

Comment:
```suggestion
        bedav
```
`bedav` with `b` small to be consistent with the icon